src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,132 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String VALID_PASSWORD = "123456789";
+    private static final int VALID_AGE = 30;
+    private static final User FIRST_USER_WITH_LOGIN_MICHAEL =
+            new User("Michael", VALID_PASSWORD, VALID_AGE);
+    private static final User NULL_USER = null;
+    private static final User INVALID_PASSWORD_USER =
+            new User("Olivia", "12345", VALID_AGE);
+    private static final User VALID_PASSWORD_USER_1 = new User("Sophia", "123456", VALID_AGE);
+    private static final User VALID_PASSWORD_USER_2 =
+            new User("Isabella", "1234567", VALID_AGE);
+    private static final User NULL_PASSWORD_USER = new User("William", null, VALID_AGE);
+    private static final User INVALID_AGE_USER = new User("Benjamin", VALID_PASSWORD, 16);
+    private static final User VALID_AGE_USER_1 = new User("Sebastian", VALID_PASSWORD, 19);
```

Comment:
Using numbers in names is a bad practice 
```suggestion
    private static final User VALID_AGE_USER_SEBASTIAN = new User("Sebastian", VALID_PASSWORD, 19);
```